src/validators/form-validator.ts

```diff
@@ -24,33 +27,55 @@ export const FieldValidator: FormFieldValidator = {
       const min = Number(field.questionOptions.min);
       const max = Number(field.questionOptions.max);
       if (isEmpty(value)) return [];
-      return !Number.isNaN(min) || !Number.isNaN(max) ? numberInputRangeValidator(min, max, value) : [];
+      return !Number.isNaN(min) || !Number.isNaN(max) ? numberInputRangeValidator(min, max, value, field) : [];
     }
     return [];
   },
 };
 
-export function numberInputRangeValidator(min: number, max: number, inputValue: number) {
+export function numberInputRangeValidator(min: number, max: number, inputValue: number, field?: FormField) {
   if (!Number.isNaN(min) && inputValue < min) {
     return [
       {
-        resultType: 'error',
+        // t('error', 'error')
+        resultType: translateFrom(moduleName, 'error', 'error'),
```

Comment:
I'm a bit curious of whether `translateFrom(moduleName, 'error', 'error')` would work here. I think the `moduleName` should point to an app that is consuming this library ie. the "@openmrs/esm-form-engine-app" or if the translations exist in core. cc: @vasharma05 @ibacher